### ISSUE TITLE
Server side task timeout format string error

### ISSUE THREAD

**Issue by aarontp** [google/turbinia#1031]
Seeing this on the Server when Tasks time out on the Server side.

```ValueError
Unknown format code 'd' for object of type 'str'
Traceback (most recent call last): File "/usr/local/bin/turbiniactl", line 33, in <module> sys.exit(load_entry_point('turbinia==20220325', 'console_scripts', 'turbiniactl')()) File "/usr/local/lib/python3.8/dist-packages/turbinia-20220325-py3.8.egg/turbinia/turbiniactl.py", line 996, in main process_args(sys.argv[1:]) File "/usr/local/lib/python3.8/dist-packages/turbinia-20220325-py3.8.egg/turbinia/turbiniactl.py", line 677, in process_args server.start() File "/usr/local/lib/python3.8/dist-packages/turbinia-20220325-py3.8.egg/turbinia/server.py", line 64, in start self.task_manager.run() File "/usr/local/lib/python3.8/dist-packages/turbinia-20220325-py3.8.egg/turbinia/task_manager.py", line 590, in run for task in self.process_tasks(): File "/usr/local/lib/python3.8/dist-packages/turbinia-20220325-py3.8.egg/turbinia/task_manager.py", line 806, in process_tasks 'Task {0:s} timed on server out after {0:d} seconds. Auto-closing Task.' ValueError: Unknown format code 'd' for object of type 'str'```

**Comment by hacktobeer** [comment]
Following the code this could only happen if `timeout` is returned in below piece of code as undefined which afaik would result in a str (instead of the expected integer). Setting this at the start check_task_timeout function would solve this.

https://github.com/google/turbinia/blob/4de58abc33f68304161803067e17dd13c7f70ab5/turbinia/task_manager.py#L315

Any other ideas?

**Comment by aarontp** [comment]
It look me longer than it should have to figure this out, but apparently I had the wrong format parameter number there :(.  I had tested with the local version, but apparently not the PSQ version.  Should be fixed in #1033 .
